### PR TITLE
feat: add categories table

### PR DESCRIPTION
**Pull request by sundriedtomato12**
## Changes
- added categories table so that we can pull out a list of categories in the models page
- changed model_group (string) to category_id (integer) for consistency
- standardised all fields in model files to snake_case
- added models controller with function to get models by categories
- added get route to get models by categories

## How to test
- sorry guys y'all will need to setup the tables and seed data again

## Known Issues
- Indicate known issues, and when they are to be fixed if they are out of scope

## Screenshots (if any)
- Screenshots of features, before/after

### CODE REVIEW

**Comment by staceyng** [comment]
gasp @sundriedtomato12 why is your editor doing single quotes. Can you check if you're using prettier and this setting in preferences is unchecked 
image

File: controllers/models.controller.mjs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      }
+    } else {
+      // this fetches models based on selected category
+      try {
+        const models = await db.Model.findAll({
+          where: {
+            category_id: categoryId
+          }
+        })
+        res.send(models)
```
Comment (staceyng) [inline controllers/models.controller.mjs:22]: Lets go with res.json 
```suggestion
        res.json({result: models})
```

**Comment by staceyng** [comment]
Seeding and logic looks good, just some small cleanups.

**Review by Nyx92 — APPROVED** [review]
ok

**Comment by staceyng** [comment]
Sad the single quotes are still happening

**Review by staceyng — APPROVED** [review]

**Pull request merged**